R/slide.R

```diff
@@ -186,20 +153,63 @@ epi_slide <- function(
 
     .f <- quosures
     # Magic value that passes zero args as dots in calls below. Equivalent to
-    # `... <- missing_arg()`, but use `assign` to avoid warning about
-    # improper use of dots.
+    # `... <- missing_arg()`, but `assign` avoids warning about improper use of
+    # dots.
     assign("...", missing_arg())
   } else {
     used_data_masking <- FALSE
   }
-
   f <- as_slide_computation(.f, ...)
 
+  align <- rlang::arg_match(.align)
+  time_type <- attr(.x, "metadata")$time_type
+  if (is.null(.window_size)) {
+    if (time_type == "week") {
+      .window_size <- as.difftime(1, units = "weeks")
+    } else {
+      .window_size <- 1
+    }
+  }
```

Comment:
issue: this default doesn't make sense, and we had problems with a default of 7 in the past.

I think issues with 7 were: we're not always using daily data, and, when we were using `n` instead of `.n`, there were some weird messages that could be produced when someone wanted to count things.